src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    divide(num) {
+      this.result /= num;
+    },
+
+    multiply(num) {
+      this.result *= num;
+    },
+
+    operate(operation, num) {
+      operation.call(this, num);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+  };
 }
```

Comment:
The function `makeCalculator` is correctly named according to the task requirements. However, the methods within the returned object can be simplified by using method shorthand syntax as per the [CODE KNOWLEDGE] checklist point.

Reply:
The methods within the returned object have already simplified.